pkg/sql/sem/builtins/builtins.go

```diff
@@ -260,6 +260,24 @@ var builtins = map[string]builtinDefinition{
 		),
 	),
 
+	"unaccent": makeBuiltin(tree.FunctionProperties{Category: categoryString},
+		stringOverload1(
+			func(evalCtx *tree.EvalContext, s string) (tree.Datum, error) {
+				separatedStrings := strings.Split(s, "")
```

Comment:
nit: this is more efficient if we use a `strings.Builder` to write out the whole string.
I feel as if we should be iterating rune by rune, something like

```
var b strings.Builder
var changed bool
for _, ch := range s {
   if repl, ok := unaccent.Characters[ch]; ok {
      b.WriteString(repl)
      changed = true
   } else {
      b.WriteRune(ch)
   }
}

ret := s
// .String() can be expensive
if changed {
   ret = b.String()
}
return tree.NewDString(ret), nil
```

Reply:
Ok, done